### DISCUSSION TITLE
Progress On Vs2022 Support

### DISCUSSION THREAD

**Discussion by TheEndHunter** [HJLebbink/asm-dude#144] [Q&A] [answered]
I Have a Few Questions;
- What is the Current Progress for Support of VS2022?
- What is left to be done to support this?
- Where can we get a Copy of the current state of progress to help test it or to contribute to it's completion?

**Comment by karen-terterian-dle** [comment]
Hi, I would also want to know if VS2022 support is in the works and if you need any help.

**Comment by henkjan-sneller** [comment]
Im currently porting everything to LSP (language server protocol), because **everything** was broken in vs2022, I have it almost working in VS, but not in VSCode yet. Once it is a LSP you can 'help' and add it to your favorite IDE.

**Reply by karen-terterian-dle** [reply]
Oh, great to hear! Thanks for your work, this extension helped me so much in the past!

**Reply by TheEndHunter** [reply]
good to hear progress is being made,  thanks for your hard work!

**Comment by henkjan-sneller** [comment, marked as answer]
for VS2022 support, see https://github.com/SnellerInc/asm-dude/discussions/1
